R/comparePlot.R

```diff
@@ -107,24 +108,24 @@ comparePlot <- function(compareDF,
                         legendPosition = "right",
                         footnote,
                         footnoteSize = 3,
-                        footnoteColor = "black"
-) {
-    set.seed(1954)
+                        footnoteColor = "black",
+                        footnoteJust = 1) {
     assertthat::assert_that(sum(apply(compareDF, 2, FUN = is.numeric)) >= 2,
-                            msg = "Need at least two numeric columns in compareDF.")
-    assertthat::assert_that(plotType %in% c("canvasXpress", "ggplot"),
-                            msg = "Plot type must be either canvasXpress or ggplot.")
-    assertthat::assert_that(length(symbolSize) == 4,
-                            length(symbolShape) == 4,
-                            length(symbolColor) == 4,
-                            length(symbolFill) == 4,
-                            msg = paste0("All specified symbol arguments must be of length 4, ",
-                                         "including symbolSize, symbolShape, symbolColor, and symbolFill."))
-
+                           msg = "Need at least two numeric columns in compareDF.")
+    plotType <- tolower(plotType)
+    assertthat::assert_that(plotType %in% c("canvasxpress", "ggplot"),
+                                       msg = "Plot type must be either canvasXpress or ggplot.")
+    assertthat::assert_that(length(symbolSize)  == 4,
```

Comment:
There are no validations for any optional parameters or plot specific parameters. Can you add them to the code. For instance, if character vector is given to symbolSize instead of a numeric vector, canvasXpress plot would crash. We need handle such failures. 
Another item that I can think of is symbol fill and symbol color both are not required for CanvasXpress. so even if symbol color is given in an incorrect format, Cxplot should still go through.

Reply:
done